path_2024_task_1.py

```diff
@@ -0,0 +1,16 @@
+'''
+Task 1 path
+'''
+
+import dronekit
+
+
+CONNECTION_ADDRESS = "tcp:localhost:14550"
+
+
+def run() -> int:
+    """
+    Reads in hardcoded waypoints from CSV file and sends drone commands.
```

Comment:
For the docstring can you change it to this: `Uploads mission to run a maximum number of laps and monitors the mission for early landing.`